app/views/mailer/comment.html.erb

```diff
@@ -18,6 +18,10 @@
   </p>
 
   <p style="font-family: 'Open Sans','Helvetica Neue',arial,sans-serif;font-size: 12px;font-weight: normal;line-height: 20px;">
-    <%= t("mailers.config.manage_email_subscriptions") %> <%= link_to t("account.show.title"), account_url, style: "color: #2895F1; text-decoration:none;" %>
+    <%= t("mailers.config.manage_email_subscriptions") %>
+    <%= link_to t("mailers.comment.unsubscribe"),
+                edit_unsubscribe_url(@receiver.unsubscribe_hash),
+                style: "color: #2895F1; text-decoration:none;",
+                target: :_blank %>
```

Comment:
Why `target: :_blank`? We almost never use it in mailer views :thinking:.

The same applies to the other links added in this pull request.